### ISSUE TITLE
[FEATURE] Autoscroll

### ISSUE THREAD

**Issue by KendallDoesCoding** [EddieHubCommunity/LinkFree#1094]
### Description

A tiny button, to auto scroll around the page and see all the coders in Linkfree

**Comment by emmalearnscode** [comment]
Can you elaborate on what you mean. Do you have an example website that does this?

**Comment by KendallDoesCoding** [comment]
> Can you elaborate on what you mean. Do you have an example website that does this?

![image](https://user-images.githubusercontent.com/84712013/153361716-cd670ea2-078d-4a9d-a30f-fa8d517ad8b9.png)
Example Site (button located down on the site): https://tabs.ultimate-guitar.com/tab/queen/dont-stop-me-now-chords-80655

**Comment by emmalearnscode** [comment]
I don't think this is needed and can imagine it's not great for accessibility. I can understand in the example that you might want the page to autoscroll so you can play the music but I don't see that as a usecase here.

**Comment by KendallDoesCoding** [comment]
I mean, this is also kinda a Coding Network? Where people can find collaborators, like I, contacted one person who was in Linkfree (Actually who made a thread in support) giving them feedback and asking for help on a small thing, so a autoscroll can be useful so it scrolls and you can see all the profiles, but yeah... I get what yall mean.... Closing!!
